dockerfiles/theia/e2e/Dockerfile

```diff
@@ -15,7 +15,7 @@ ENV NOCDN=true
 
 RUN printf "deb http://archive.debian.org/debian/ jessie main\ndeb-src http://archive.debian.org/debian/ jessie main\ndeb http://security.debian.org jessie/updates main\ndeb-src http://security.debian.org jessie/updates main" > /etc/apt/sources.list
 RUN apt-get update && \
-    apt-get install -y libx11-dev libxkbfile-dev
+    apt-get install -y libx11-dev libxkbfile-dev sudo iproute2
```

Comment:
why sudo? AFAIK it's not used

Reply:
e2e image copies /entrypoint.sh from `theia` image. `sudo` is used there.

https://github.com/PizzaFactory/che-theia/blob/4f92c61fde7b9c09757fcd83cec40d9694b14ed2/dockerfiles/theia/src/entrypoint.sh#L33-L34

The script checks if `sudo` is usable. So it won't make bad effects to Openshift based instances.

Reply:
And it's better to apply https://github.com/eclipse/che-theia/pull/292 after merged this patch.
`che-theia` should be checked to run both root and non-root env.

Reply:
I would separate the PRs

One for the 'short wait time' upon boot
one around permissions which is partially handled in another PR

Reply:
I see. I left the timeout issue here. And the sudo issue is moved to #293